### PR TITLE
Improve header design

### PR DESCRIPTION
**Pull request by filfreire**
Closes issue #15 

You can check how it looks like on http://filfreire.com/portotestersmeetup.github.io/

@FabioCB @filipemcarvalho FYI

### CODE REVIEW

**Comment by filfreire** [comment]
@FabioCB @filipemcarvalho please CR.

File: assets/css/style.scss [hunk 1]
```diff
@@ -16,11 +16,11 @@ h1.project-name:after{
 }
 
 .page-header {
-  background-image:url("../../images/cover/lineCover.png");
   background-repeat: no-repeat;
   background-position: center center;
-  background-color: lightblue;
+  background-color: #02afe8;;
```
Comment (filipemcarvalho) [inline assets/css/style.scss]: Are the two ;; on purpose here?

**Review by filipemcarvalho — COMMENTED** [review]
Thank you for the contribution 👍 LGTM. Just did a small comment, can you please check and address if needed?

File: assets/css/style.scss (hunk 1 above)
Comment (filfreire) [reply assets/css/style.scss]: @filipemcarvalho fixed, my bad 👍

**Comment by filfreire** [comment]
@filipemcarvalho done

**Review by filipemcarvalho — APPROVED** [review]

**Pull request merged**